core/services/keeper/delegate.go

```diff
@@ -28,8 +31,7 @@ type Delegate struct {
 	logBroadcaster  log.Broadcaster
 }
 
-var _ job.Delegate = (*Delegate)(nil)
```

Comment:
This pattern is a way to make sure your Delegate conforms to the `job.Delegate` interface, i.e. it's a way to get "nominative types" in go, which only has structural types. I suggest keeping it in!

Reply:
looks like it was just moved